packages/components/Menu/src/MenuItemCheckbox/MenuItemCheckbox.styling.ts

```diff
@@ -7,6 +9,7 @@ import { menuItemCheckboxName } from './MenuItemCheckbox.types';
 import { defaultMenuItemCheckboxTokens } from './MenuItemCheckboxTokens';
 
 export const menuItemCheckboxStates: (keyof MenuItemCheckboxTokens)[] = ['hovered', 'focused', 'pressed', 'disabled', 'checked'];
+const hasPresetRententionForA11y = Platform.OS === 'android';
```

Comment:
Does does need to be "`pressRetnetion`"?

Reply:
good catch 👍   Changed.